x-pack/plugins/ml/public/application/contexts/kibana/use_ml_license.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import useObservable from 'react-use/lib/useObservable';
+import { useMlKibana } from './kibana_context';
+
+export const useMlLicenseInfo = () => {
```

Comment:
this file might be better located in `x-pack/plugins/ml/public/application/license`

Reply:
I was thinking about it, but the service lives in the kibana context, so I reckon it also makes sense